index.html

```diff
@@ -8,8 +8,8 @@
     <link rel="stylesheet" href="index.css">
     <title>Document</title>
 </head>
-<body>
-    <main id="main">
+<body style="background-image: url(https://www.reddit.com/r/csshelp/comments/rf1ppz/why_add_before_in_this_situation/);">
```

Comment:
Please keep in mind the html&css best practices. I would suggest that you avoid using inline styling. This helps keep code clean and also enables debugging easier.